src/components/ChattingScreen.js

```diff
@@ -1,31 +1,136 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
 import Header from './Header';
 import MessageSender from './MessageSender';
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
-
-	const MSGLIST = [
-		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
+  const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
+
+  const MSGLIST = [
+    { user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
+    { user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
+    { user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
+    { user: false, content: '안녕 은아' },
+    { user: false, content: 'ㅁㅎ?' },
+    { user: true, content: '시원아 넌 최고의 팀장이야' },
+    { user: false, content: '나도 아니까  ' },
+    { user: false, content: '그만 말해줘도 돼' },
+    { user: true, content: 'ㅠㅠ' },
+  ];
+
+  const [currentUser, setCurrentUser] = useState(false); // 현재 채팅하는 사람
+  const [newMSGList, setNewMSGList] = useState(MSGLIST); // Re-Rendering하기 위하여 생성
+
+  useEffect(() => {
+    window.scrollBy(0, window.innerHeight);
+  });
+
+  const addMessageToList = (message) => {
+    setNewMSGList(
+      newMSGList.concat([
+        {
+          user: currentUser,
+          content: message,
+        },
+      ])
+    );
+  };
+
+  const changeUser = () => {
+    setCurrentUser(!currentUser);
+  };
+
+  const chattingList = newMSGList.map((msg, index) => {
+    return (
+      <MessageLine key={index} user={msg.user} profileImg={msg.user ? EUNKO : COOL} message={msg.content}></MessageLine>
+    );
+  });
+
+  return (
+    <Wrapper>
+      <Header
+        user={currentUser ? '고은' : '정쿨'}
+        profileImg={currentUser ? EUNKO : COOL}
+        changeUser={changeUser}
```

Comment:
props 이름과 변수의 이름이 같다면 spread operator를 이용해 전달하는 방법도 있답니당!
```suggestion
        {...{changeUser}}
```

Reply:
옹 멋있어요

Reply:
spread operator 메모..